torq.q

```diff
@@ -583,11 +588,22 @@ if[@[value;`.ps.loaded;0b]; .ps.initialise[]]
 // initialise connections
 if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
+// function to execute functions in .proc.initlist
+.proc.init:{
+	if[count .proc.initlist;
+	{[a]@[value;a;{[x;a].lg.e[`init;"unable to run initialisation function for ",a," - ",x," error"]}[;a]]}
```

Comment:
I think you need to indent the condition

Will this actually work?  What if a is a function like (`myfunc;1;2) - surely the error trap will fail as it can't print it?  I think you need to -3! it. 

Also can you add more logging.   For each initialisation function that runs we need a log line 

"running initialisation function: ",-3!a

If we don't have any to run then we need a log line to say that, e.g. "no initialisation functions found"